### PR TITLE
Script to test FFT and Filter

### PR DESCRIPTION
**Pull request by mouryarahul**
This repo contains the script to test:

1. ulab.convolve

1. ulab.fft.fft

1. ulab.fft.ifft

1. ulab.fft.spectrogram

### CODE REVIEW

**Review by v923z — CHANGES_REQUESTED** [review]

**Review by mouryarahul — COMMENTED** [review]
I didn't get you. Do you mean we do not need to test `interp` and other functions included in this test?

File: tests/numpy/filter_fft.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    use_ulab = True
+except ImportError:
+    import numpy as np
+    use_ulab = False
+
+np.set_printoptions(threshold=8, edgeitems=4)
+x = np.array((1,2,3))
+y = np.array((1,10,100,1000))
+
+result = (np.convolve(x, y))
```
[the remark this answers is not in the document]
Comment (mouryarahul) [reply tests/numpy/filter_fft.py:13]: Okay, I will separate them.

**Comment by v923z** [comment]
> I didn't get you. Do you mean we do not need to test `interp` and other functions included in this test?

By my mind, each test should test only a single function, or a set of closely related functions. Even more so that `circuitpython` defines sub-modules, and `interp` and `fft` are not in the same one.

File: tests/numpy/filter_fft.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (v923z) [reply tests/numpy/filter_fft.py:13]: Thanks!

**Comment by mouryarahul** [comment]
> > I didn't get you. Do you mean we do not need to test `interp` and other functions included in this test?
> 
> By my mind, each test should test only a single function, or a set of closely related functions. Even more so that `circuitpython` defines sub-modules, and `interp` and `fft` are not in the same one.

@v923z Got it now :). I'll separate them.

**Comment by v923z** [comment]
@mouryarahul  Thanks for sticking with it will the very end!

**Pull request merged**